ciao-cli/image.go

```diff
@@ -36,17 +35,8 @@ const imageTemplateDesc = `struct {
 	SizeBytes        int      // Size of image in bytes
 	ID               string   // Image UUID
 	Status           string   // Image status.  Can be queued or active
-	Owner            string   // Tenant that owns the image
-	ContainerFormat  string   // Format of the container, e.g., ovf
-	DiskFormat       string   // Format of the image, e.g., qcow2
-	MinDiskGigabytes int      // Minimum amount of disk space required to boot image
-	MinRAMMegabytes  int      // Minimum amount of RAM required to boot image
-	Checksum         string   // Checksum of image data
-	Protected        bool     // Indicates whether or not an image can be deleted
-	Visibility       string   // Indicates whether the image is public or private
```

Comment:
Visibility is useful to have information about which images are private (self created images) and public (all-tenants available) images.